src/firefly/js/templates/hydra/HydraViewer.jsx

```diff
@@ -80,13 +86,17 @@ HydraViewer.defaultProps = {
     appTitle: 'Time Series Viewer'
 };
 
-function onReady(menu) {
+function onReady(menu, appTitle) {
     dispatchSetMenu({menuItems: makeMenuItems(menu)});
 
     const {hasImages, hasTables, hasXyPlots} = getLayouInfo();
     if (!(hasImages || hasTables || hasXyPlots)) {
         const goto = getActionFromUrl() || {type: SHOW_DROPDOWN};
-        if (goto) flux.process(goto);
+        if (goto) {
+            flux.process(goto);
+            // if app didn't start with Results view, app hint for tabs menu is not needed
+            goto?.type === SHOW_DROPDOWN && dispatchAddPreference(appHintPrefName(appTitle, APP_HINT_IDS.TABS_MENU), false);
+        }
```

Comment:
Now that we are using `item.landing` as a way to define where the application should 'goto', should we duplicate the same logic here?  Better yet, make a reusable function for it.

Reply:
Again, I meant to ask this - is landing a boolean property supposed to be defined on menu items?